init.lua

```diff
@@ -90,9 +75,10 @@ minetest.register_chatcommand("choosecase", {
 	func = function(name, params)
 		local lname = params:lower()
 		local worldpath = minetest.get_worldpath()
-		for iname, data in pairs(minetest.auth_table) do
+		local auth = minetest.get_auth_handler()
+		for iname in pairs(auth.iterate) do
```

Comment:
Have you tested this? It should be something like   `for inane in auth.iterate() do`

Pairs *creates* an iterator, it doesn't iterate over one

Reply:
forgot parenthesis, added and tested, seems to work

Reply:
Have you tested this with default authentication?

Reply:
no, my mistake was having sauth enabled